packages/bodiless-ui/src/elements.tsx

```diff
@@ -128,7 +128,7 @@ export const ComponentFormUnwrapButton = addClasses(
 export const ComponentFormCloseButton = flow(
   addClasses('hover:bl-text-red bl-float-right'),
   removeClasses('hover:bl-text-green'),
-  withChild(() => <Icon>cancel</Icon>),
+  withChild(() => <Icon data-close-button>cancel</Icon>),
```

Comment:
Moved to `/Bodiless-JS/packages/bodiless-core/src/contextMenuForm.tsx` into `FormChromeBase`. I gave it an `id` instead of `data-*` attr. `element.closest()` works way faster with `id` than `data-*` attrs.